resource-timing/embed-navigate.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>Resource Timing embed navigate</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/nested-contexts.js"></script>
```

Comment:
Add `<div id=log></div>` here so the `<embed>` doesn't show up at the top?

Reply:
cool